src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -29,5 +30,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('sylius_api.enabled', $config['enabled']);
 
         $loader->load('services.xml');
+
+        if (!$container->hasParameter('enable_swagger') || !$container->getParameter('enable_swagger')) {
+            return;
+        }
+
```

Comment:
It should be api_platform.enable_swagger or it should not be in api_platform namespace in config.yaml modified in this PR.